api/system-settings/getOtpToken

```diff
@@ -23,25 +23,34 @@
 use warnings;
```

Comment:
Cosmetic: please do not use filename with camel case.
See all other helpers. The name should look like: get-otp-token